services/movie_session.py

```diff
@@ -0,0 +1,58 @@
+from db.models import MovieSession, CinemaHall, Movie
+from datetime import datetime as dt
+
+
+def create_movie_session(
+        movie_show_time: dt,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+
+    hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> list[MovieSession]:
+    date = dt.strptime(session_date, "%Y-%m-%d") if session_date else None
+    if date:
+        return MovieSession.objects.filter(show_time__date=date).all()
+    return MovieSession.objects.all()
```

Comment:
not need to create `date` variable

```suggestion
    if session_date:
         return MovieSession.objects.filter(show_time__date=session_date)
     return MovieSession.objects.all()
```